Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/KeyboardTypeModifier.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  KeyboardTypeModifier.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/18/23.
+//
+
+import SwiftUI
+
+/// Sets the keyboard type for this view.
+///
+/// ```html
+/// <TextField modifiers={keyboard_type(@native, type: :email_address)}>
+///     Email
+/// </TextField>
+/// ```
+///
+/// ## Arguments
+/// * ``type``
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+struct KeyboardTypeModifier: ViewModifier, Decodable, Equatable {
```

Comment:
This is missing availability checks, since the modifier is only available on iOS/tvOS. CI is probably failing for this reason.